test_runner/src/main/kotlin/ftl/run/exception/ExceptionHandler.kt

```diff
@@ -54,7 +55,7 @@ internal fun withGlobalExceptionHandling(block: () -> Int, exitProcessFunction:
             }
 
             is InfrastructureError -> {
```

Comment:
Do we want to change debug level for `FTLError` as well?

Reply:
I am not sure, does FTLError occurs only on Google Api issues?

Reply:
According to error description, it's thrown when an internal error occurs.
So I guess you are right...

Reply:
we could leave it for now and see later how it is working